.drone.yml

```diff
@@ -16,6 +17,6 @@ publish:
     tags:
       - $(git rev-parse --short HEAD)
     when:
-      branch: master
+      branch: fix-broken-build
```

Comment:
And by file, I mean line. This one, to be particular. Thanks, github. :facepalm: